### PR TITLE
[DEBUG] Add some verbose logging

### PR DESCRIPTION
**Pull request by lackstein**

### CODE REVIEW

File: library/Faktory/Worker.hs [hunk 1]
```diff
@@ -116,12 +118,22 @@ startWorker settings workerSettings handler = do
...
+              )
+                `catchAny` ( \e -> do
+                              now <- getZonedTime
+                              liftIO $ putStrLn $ show now <> " - EXCEPTION: " <> displayException e
+                              throwIO e
+                           )
+            )
+            ( do
+                now <- getZonedTime
+                putStrLn $ show now <> " - KILLING HEARTBEAT"
```
Comment (elldritch) [inline library/Faktory/Worker.hs:134]: You use this helper enough that it's probably worth abstracting.

**Review by elldritch — APPROVED** [review]
Logging functionality LGTM, but I'd prefer if you could make this change without all the style changes. The style changes will make it harder for us to upstream these changes later.

**Comment by lackstein** [comment]
@elldritch I'm not planning to merge this in. If we think it's helpful past debugging this issue, we should modify the library to use our logging system.

**Comment by elldritch** [comment]
@lackstein we good to close this?

**Pull request closed without merging**